packages/dev-utils/src/devApp/render.tsx

```diff
@@ -193,6 +209,18 @@ export class DevAppBuilder {
       apis,
       plugins: this.plugins,
       themes: this.themes,
+      components: {
+        SignInPage: props => {
+          return (
+            <SignInPage
+              {...props}
+              providers={['guest', 'custom', ...this.signInProviders]}
```

Comment:
Let's skip `custom` imo, it's just noise 😅

Reply:
Yeah, never figured out how to use it 🤣